library/build.gradle.kts

```diff
@@ -1,22 +1,19 @@
 
-val ktorVersion = "1.5.0"
-val coroutineVersion = "1.4.2"
+val ktorVersion = "1.5.3"
+val coroutineVersion = "1.4.3"
 
 plugins {
     kotlin("multiplatform")
     id("com.android.library")
     id("kotlinx-serialization")
     id("kotlin-android-extensions")
+    id("com.chromaticnoise.multiplatform-swiftpackage") version "2.0.3"
     id("maven-publish")
 }
 
 group = "edu.usf.cutr.otp"
 version = "1.0.0"
 
-repositories {
```

Comment:
@BumbleFlash See https://github.com/CUTR-at-USF/opentripplanner-client-library/pull/13/files#r609862320. Can we get rid of this?

Reply:
Yes, I think you can. I'm not getting any local dependencies so I don't think we need it.